index.html

```diff
@@ -6,15 +6,47 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="description"
-  content="full description of the app">
+        content="Software Engineer, Graphic Designer, Producer, Ramoun helps you remove obstcles to achieve your goals. Specializing in Full Stack Web Development, Logo Design & Music Producing">
   <meta name="robot" content="index,follow">
-  <link rel="stylesheet" href="styles/reset.css">
-  <link rel="stylesheet" href="styles/debug.css">
-  <title>Document | Description</title>
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/layout.css">
+  <link rel="stylesheet" href="css/-debug.css">
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <title>Ramoun | Software Engineer</title>
 </head>
 
 <body>
-  <h1>Hello, World!</h1>
+  <!-- header/begin -->
+  <header class="main-header bg">
+    <div class="wrapper">
+      <div id="logo">
+        <img src="./images/darwin.jpg" alt="">
+        <a href="./">Ramoun</a>
+      </div>
```

Comment:
- [ ] [OPTIONAL] This was a very good idea to have the logo as an anchor tag `<a>`, I think this could be even better if you could just take also the image inside the `<a>` tag because people may take the image as part of the logo and try to click on it. you may have something like this instead 
```
<a href="#" class="logo">
    <img src="./images/darwin.jpg" alt="">
    <span>Ramoun</span>
</a>
```
With this, by clicking on the image we should have the same behavior as clicking on the name. 😉